internal/options/kube.go

```diff
@@ -20,11 +20,8 @@ type kubeOpts struct {
 	config    *rest.Config
 }
 
-func NewKube(controlPlaneURL string, groupName string, claimName string, config *rest.Config) (ListenerOpts, error) {
+func NewKube(groupName string, claimName string, config *rest.Config) (ListenerOpts, error) {
```

Comment:
I don't want to be pedantic but can I ask you to split changes into several commits?

One for the Helm charts, another for the Go code-base changes.

Reply:
Yep, but I'm not sure if we should do this!